ParabolInc/parabol
# feat: domain stats analytics

Base: https://github.com/ParabolInc/parabol/pull/6687

Fix https://github.com/ParabolInc/parabol/issues/6681

This PR sends an event to Segment when the viewer visits the `/usage` page and when they click the domain stats CTA. 

### To test

- Copy & paste the `SEGMENT_WRITE_KEY` and `SEGMENT_FN_KEY` from production in the devops repo
- Comment out the `PRODUCTION` check in `Analytics.ts`
- Visit the `/usage` page and click on the CTA
- Check the network tab of the dev tools and see that the `SendClientSegmentEventMutation` was executed
- Visit Amplitude, search for your `viewerId`, and see that the events were recorded
